frontend/src/pages/profile/MemberDetails.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+import { Accordion, AccordionHeader, AccordionBody } from '@material-tailwind/react';
+import MemberDetailsSection from './MemberDetailsSection';
+import type { Personnel } from '../dashboard';
+import dayjs from 'dayjs';
+
+function ArrowIcon({ id, open }: { id: number; open: number }) {
```

Comment:
Split into Icons file in the UI directory?

Reply:
Using hero icons now